src/js/pages/catalog/DeployFrameworkConfiguration.js

```diff
@@ -0,0 +1,144 @@
+import React, { PropTypes } from "react";
+import mixin from "reactjs-mixin";
+import qs from "query-string";
+import deepEqual from "deep-equal";
+import { StoreMixin } from "mesosphere-shared-reactjs";
+import { routerShape } from "react-router";
+import CosmosPackagesStore from "#SRC/js/stores/CosmosPackagesStore";
+import Util from "#SRC/js/utils/Util";
+import FrameworkConfiguration from "#SRC/js/components/FrameworkConfiguration";
+import Loader from "#SRC/js/components/Loader";
+
+export default class DeployFrameworkConfiguration extends mixin(StoreMixin) {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      packageDetails: null,
+      deployErrors: null,
+      formErrors: {},
+      formData: null
+    };
+
+    this.store_listeners = [
+      {
+        name: "cosmosPackages",
+        events: [
+          "packageDescriptionSuccess",
+          "packageDescriptionError",
+          "installSuccess",
+          "installError"
+        ]
+      }
+    ];
+
+    CosmosPackagesStore.fetchPackageDescription(
+      props.params.packageName,
+      props.location.query.version
+    );
+  }
+
+  onCosmosPackagesStorePackageDescriptionSuccess() {
+    const packageDetails = CosmosPackagesStore.getPackageDetails();
+    const formData = this.initializeFormDataFromSchema(packageDetails.config);
+    this.setState({ packageDetails, formData });
+  }
+
+  onCosmosPackagesStorePackageDescriptionError() {
+    // todo figure out what this response looks like and what we should do
```

Comment:
Good that you haven't just _omitted_ the method! 
If you're not going to resolve this TODO, while working on this PR, please create a jira and put it's number here.